lib/authorization/assertion/twitter_auth.rb

```diff
@@ -0,0 +1,49 @@
+module Authorization
+  module Assertion
+    class TwitterAuth
+      def initialize(access_token, access_token_secret)
+        @access_token = access_token
+        @access_token_secret = access_token_secret
+        @twitter_client = twitter_client
```

Comment:
Instead of saving this to an instance variable in the `initialize` method and then accessing that, it might be better to memoize it within the `twitter_client` method and refer to the method, like so:

`@twitter_client ||= Twitter::REST::Client.new do ...`